src/pset/map/global.rs

```diff
@@ -236,13 +244,23 @@ impl Map for Global {
 
         // Serializing version only for non-default value; otherwise test vectors fail
         if self.version > 0 {
+            let ver = self.version; //hack to use macro
+            impl_pset_get_pair!(
+                rv.push_mandatory(ver as <PSET_GLOBAL_VERSION, _>)
+            )
+        }
```

Comment:
Now that this is a mandatory field, we should serialize it unconditionally I think.

Reply:
It's only a mandatory field in v2

Reply:
But there are no other versions of PSET.